tests/outputs/a_manurev/refereeReports.yaml

```diff
@@ -0,0 +1,44 @@
+1902nature:
+  claimed_found_what:
+    - gravity waves
+  claimed_why_important:
+    - more money for ice cream
+  did_how:
+    - measured with a ruler
+  did_what:
+    - found a much cheaper way to measure gravity waves
+  due_date: '2020-04-11'
+  editor_eyes_only: to be honest, I don't believe a word of it
+  final_assessment:
+    - The authors should really start over
+  first_author_last_name: Wingit
+  freewrite: this comment didn't fit anywhere above
+  journal: Nature
+  month: 2
+  recommendation: invited
+  requester: Max Planck
+  reviewer: sbillinge
+  status: submitted
+  title: a ruler approach to measuring gravity waves
+  validity_assessment:
+    - complete rubbish
+  year: 2019
+2007_einstein_a:
+  claimed_found_what: []
+  claimed_why_important: []
+  did_how: []
+  did_what: []
+  due_date: '2020-09-15'
+  editor_eyes_only: ''
+  final_assessment: []
+  first_author_last_name: Einstein
+  freewrite: ''
+  journal: ''
```

Comment:
I would like journal to be a required argument and be filled in.

Reply:
Complete. Now it is a required field in the command line.